.github/workflows/ci.yml

```diff
@@ -131,20 +146,23 @@ jobs:
       - name: Freeze
         run: cabal freeze --project-file=cabal.static.project
 
-      - uses: actions/cache@v4
+      - name: Cache
+        uses: actions/cache@v4
         with:
           path: ${{ steps.setup-haskell.outputs.cabal-store }}
           key: ${{ runner.os }}-ghc-${{ steps.setup.outputs.ghc-version }}-${{ hashFiles('**/plan.json') }}
-          restore-keys: ${{ runner.os }}-${{ steps.setup.outputs.ghc-version }}-
+          restore-keys: ${{ runner.os }}-ghc-${{ steps.setup.outputs.ghc-version }}-
 
       - name: Build
         run: cabal build --project-file=cabal.static.project
 
       - name: Install
         run: |
-          bin=$(cabal -v0 --project-file=cabal.static.project list-bin print-api)
           mkdir distribution
-          cp ${bin} distribution/print-api
+          for executable in 'print-api' 'print-api-${{ matrix.ghc }}'; do
+            path=$(cabal -v0 --project-file=cabal.static.project list-bin "print-api:exe:${executable}")
+            cp "${path}" distribution
+          done
```

Comment:
Every matrix job builds two executables: The `print-api` wrapper and the `print-api-<GHC VERSION>` executable.
The former one needs to be built and distributed by _some_ matrix job, but I am not sure how to decide on one.
Hence I decided to simply build and distribute it in all jobs.
An alternative solution would be to modify `get-tested` to provide more information:
For example, given `tested-with: GHC ==9.4.8, GHC ==9.10.1`, it could add a `newest` field in each matrix job indicating whether the job uses the newest GHC in the list:
```yaml
matrix:
  include:
    - ghc: "9.4.8"
      newest: false
    - ghc: "9.10.1"
      newest: true
```
Then we could use that information to built and distribute the `print-api` wrapper executable in job for the newest GHC only.

Reply:
Ok, let's merge this and see how to proceed later on.